src/codeu/chat/util/encryption/PublicKey.java

```diff
@@ -0,0 +1,13 @@
+import java.math.BigInteger;
+
+public class PublicKey {
+    BigInteger n, e;
+
+    public PublicKey(BigInteger otherN, BigInteger otherE) {
+        n = otherN;
+        e = otherE;
+    }
+
+    public BigInteger get_n() {return n;}
```

Comment:
Stick with a single naming convention for all method and variable names. In this case getE and getN would be good. Expanding E and N would be even better.